packages/apps/transfer/src/pages/cross-chain-transfer-finisher/index.tsx

```diff
@@ -0,0 +1,129 @@
+import { Button, TextField } from '@kadena/react-components';
+
+import MainLayout from '../../components/Common/Layout/MainLayout';
+
+import {
+  StyledAccountForm,
+  StyledCheckbox,
+  StyledCheckboxLabel,
+  StyledFieldCheckbox,
+  StyledForm,
+  StyledFormButton,
+  StyledMainContent,
+  StyledToggleContainer,
+} from './styles';
+
+import { SidebarMenu } from '@/components/Global';
+import { useAppContext } from '@/context/app-context';
+import React, { FC, useState } from 'react';
+
+const CrossChainTransferFinisher: FC = () => {
+  const { network } = useAppContext();
+
+  const chainServerText =
```

Comment:
it'll be nicer if you use object lookup

`{
mainnet: {server: "", text: ""},
testnet: {server: "", text: ""},
}
//
net[network.toString()]
`